<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora de matrices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralwidget_layout" stretch="0">
    <item>
     <widget class="QWidget" name="table_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
       <item>
        <widget class="QFrame" name="table_buttons_frame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="table_buttons_horizontal_layout" stretch="5,5,100">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="table_row_spinbox_widget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="row_spinbox_vertical_layout">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">border: 0</string>
               </property>
               <property name="text">
                <string>Filas</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="row_spinbox">
               <property name="styleSheet">
                <string notr="true">border-color: rgb(0, 0, 0);</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="table_column_spinbox_widget" native="true">
            <layout class="QVBoxLayout" name="column_spinbox_vertical_layout">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
              <widget class="QLabel" name="column_spinbox_label">
               <property name="text">
                <string>Columnas</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QSpinBox" name="column_spinbox"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="table_grid_options">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="1">
             <widget class="QPushButton" name="table_fill_0_button">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <property name="text">
               <string>Rellenar espacios con 0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="table_clean_matrix_button">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa;
}</string>
              </property>
              <property name="text">
               <string>Limpiar Matriz</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="table_random_matrix_button">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <property name="text">
               <string>Matriz aleatoria</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="table_update_button">
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <property name="text">
               <string>Actualizar</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="table_export_matrix_button">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <property name="text">
               <string>Exportar Matriz</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QComboBox" name="table_solution_matrix_combobox">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid black;
border-radius: 5px
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <item>
               <property name="text">
                <string>Resolver por</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Reducci√≥n</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QPushButton" name="table_solve_matrix_button">
              <property name="styleSheet">
               <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 5px;
}
QPushButton::hover{
background-color: #eee;
}
QPushButton::pressed{
background-color: #aaa
}</string>
              </property>
              <property name="text">
               <string>Resolver</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="input_table">
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>X1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>b</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
